open-science-projects.md

```diff
@@ -23,34 +23,58 @@ Discussions on errors, new features, and other specific issues for riboviz, take
 
 # The student perspective (Emma MacKenzie and Sophie Winterbourne)
 
-## Open science projects mean less anxiety about sharing information
+## Projects available to students can be more or less open
```

Comment:
I like this section as it gives a good overview of the projects however I think the title could potentially be changed, as `Projects available to students can be more or less open` seems slightly wordy and doesn't quite describe the section.

It could potentially be changed to `Overview of available projects` or something similar

Reply:
Edited, thanks.